channelfinder/CFDataTypes.py

```diff
@@ -9,13 +9,9 @@
 @author: shroffk
```

Comment:
This change, removing support for py2, should really be highlighted in the MR, and should maybe even be its own MR